datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/UpdateNameResolver.java

```diff
@@ -62,7 +63,7 @@ private Boolean updateGlossaryTermName(
       UpdateNameInput input,
       QueryContext context
   ) {
-    if (AuthorizationUtils.canManageGlossaries(context)) {
+    if (GlossaryUtils.canManageGlossaries(context)) {
```

Comment:
So in order to change the name of the entity I've created, I might need "manage glossaries"? What if we just revert back to the same check we ran before around whether we can manage the node?

Reply:
yup definitely like this more. if you can delete the entity you should be able to rename it...